default/python/README.md

```diff
@@ -23,64 +23,37 @@ editing options.py and universe_tables.py, both of which have more information
 in comments over there. The latter, specifically, controls which star types,
 planet types, planet sizes, and also other content get placed.
 
-# Generating skeletons
-API to fetch data/read content is created in runtime using C++, 
-in order to have autocomplete in IDE skeletons (*.pyi) were generated.
-
-Skeletons can be generated in runtime environment only.  
-
-It is highly recomended to configure game to use AI script from the repo
-https://www.freeorion.org/index.php/Python_Development#Deploying_code
-
-```sh
-freeorion --ai-config freeorion\default\python\handlers\inspect_interface_config.ini 
-```
-`freeorion` is a path to the game binary (freeorion.exe on Windows).
-`freeorion\default\python\handlers\inspect_interface_config.ini` path to file in this repository. Note that relative pathes may not work, better specify an absolute path.
-
-
-Starting new game and wait until it exits to main menu with an error.
-
-Generated files will be saved to AI folder:
-- freeorion\default\python\freeorion.pyi
-- freeorion\default\python\AI\freeOrionAIInterface.pyi
-You can check full paths in AI_1.log and freeoriond.log after `Skeleton written to`.  
-
-Copy them to repository with replace, commit.
-
-# Code style check
-Each PR will be checked automatically, but you still can run checks manually.
-Fixing all issues are mandatory before merging PR.
-
-We use [flake8](https://pypi.python.org/pypi/flake8) for code style checks.
-Settings for flake8 located in the `tox.ini`.
-
-We use [black](https://pypi.org/project/black/) for code formatting.
-Settings for black are located in `pyproject.toml`.
-
-## Install dependencies
-
-```sh
-pip install flake8==3.7.9
-pip install black
-```
-
-## Format python files
-Black can be run on the top level directory:
-
-```sh
-black .
-```
-
-## Run checks
-Flake should be run from directory where `tox.ini` located
-
+# Development
+
+## Setup environment
+### Run game with AI files from the repository folder
+- Run game
+- Open Options
+- Scroll left to Others
+- Click generate persistent config
+- Check `Congig and log files path` value
+- Open directory with config
+- Edit `persistent_config.xml`
+- ```xml
+  <?xml version="1.0"?>
+  <XMLDoc>
+      <resource>
+      <path>...repo_path\default</path>
+      <stringtable>
```

Comment:
Indent of <ressource> is wrong.

The generated persistent_config.xml contains a lot, but no <resource> tag. Bug in the generation code?
I guess this does not mean to remove all other values.

Reply:
It merged with an override to the config.  Config is overridden on game install.